src/locales/_build/src/components/MainMenu/MainMenu.tsx.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
We should get rid of this file and add `src/locales/_build` to the `.gitignore` file.